doc/quick-start/Dockerfile

```diff
@@ -5,10 +5,12 @@ WORKDIR /build
 COPY . .
 
 RUN set -eux && \
-  apt-get update && apt-get install -y git build-essential autoconf autogen libtool clang && \
+  sed -i s@/deb.debian.org/@/mirrors.aliyun.com/@g /etc/apt/sources.list && \
```

Comment:
I guess it works slower on Github Action runner machines.

Reply:
But it works faster for domestic users who need to build the Xline docker image from scratch.